pos_order_mgmt/models/pos_config.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 GRAP - Sylvain LE GAL
+# Copyright 2018 Tecnativa S.L. - David Vidal
+# Copyright 2019 Coop IT Easy SCRLfs
+#                   Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    iface_reprint_done_order = fields.Boolean(
+        string='Reprint Done Orders',
+        default=True,
+        help='Allows to reprint already done orders in the frontend',
+        oldname='iface_load_done_order',
+    )
+
+    iface_return_done_order = fields.Boolean(
+        string='Return Done Orders',
+        default=True,
+        help='Allows to return already done orders in the frontend',
+        oldname='iface_load_done_order',
```

Comment:
there are two field with the same oldname value. 'iface_load_done_order'. why ?

Reply:
the field iface_load_done_order is splitted, so if you used it, you want both the functions once migrated